pom.xml

```diff
@@ -172,7 +172,7 @@
       <dependency>
         <groupId>com.google.cloud.datastore</groupId>
         <artifactId>datastore-v1-proto-client</artifactId>
-        <version>1.6.3</version>
+        <version>1.6.4-SNAPSHOT</version><!-- {x-version-update:datastore-v1-proto-client:current} -->
         <exclusions>
```

Comment:
You may be able to remove this exclusion now that the protos are from this build instead of from maven central.